caffe2/python/operator_test/elementwise_ops_test.py

```diff
@@ -128,6 +128,49 @@ def sqr_op(X):
         self.assertGradientChecks(
             gc, op, [X], 0, [0], stepsize=1e-4, threshold=1e-2)
 
+    @given(n=st.integers(5, 6), m=st.integers(4, 6), **hu.gcs)
+    def test_sqrt(self, n, m, gc, dc):
+        X = np.random.rand(n, m).astype(np.float32)
```

Comment:
Since we are doing random generation, usually it helps to generate a seed from hypothesis and explicitly set that seed.